### PR TITLE
[FE] feature: Create login page [DELETE PR]

### PR DESCRIPTION
**Pull request by joseph-robles**
First iteration of the login page (Front End WIP).

### CODE REVIEW

File: frontend/ft-social/.gitignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Logs
```
Comment (eds000n) [inline frontend/ft-social/.gitignore:1]: we don't need this as there's a .gitignore in the parent folder

File: frontend/ft-social/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# React + TypeScript + Vite
```
Comment (eds000n) [inline frontend/ft-social/README.md:1]: have you discussed as a *team* about the technologies of the project? why using vite?

**Review by eds000n — COMMENTED** [review]
there are colliding PR, you haven't discussed as a team on the project's structure. Also PR has to be done against `develop` branch

**Comment by joseph-robles** [comment]
This Pull Request will soon probably be removed and another one will be created for my new branch [FE_feat_LoginPage](https://github.com/unmsmfisi-socialapplication/social_app/tree/FE_feat_LoginPage) 
image

**Comment by joseph-robles** [comment]
New pull request #187 has been created. This PR has now been closed.

**Pull request closed without merging**